### ISSUE TITLE
Javadoc: Spacing not preserved

### ISSUE THREAD

**Issue by dhardtke** [jundl77/auto-format#2]
Hey,

I started using the library and noticed whitespace in Javadoc is not being preserved:
```java
/**
 * This Javadoc is just a stub.
 */
public class SecondExercise {
    /**
     * Method that returns the corresponding char-Array for a given input String.
     */
    char[] getCharArrayForInput(String input) {
        return null;
    }

    /**
     * Method that always returns "World".
     */
    String getWorld() {
        return null;
    }
}
```

becomes

```java
/**
* This Javadoc is just a stub.
*/
public class SecondExercise {
    /**
    * Method that returns the corresponding char-Array for a given input String.
    */
    char[] getCharArrayForInput(String input) {
        return null;
    }
    
    /**
    * Method that always returns "World".
    */
    String getWorld() {
        return null;
    }
}
```

Apart from that it's a great library!

**Comment by jundl77** [comment]
Yeah good point, so many little edge cases to think about. I'll fix it in the next couple days.

**Comment by jundl77** [comment]
The issue should be fixed in version 1.1.1 on NPM. 

Thanks for pointing that out, as well as for the kind feedback :)

**Comment by dhardtke** [comment]
It works perfectly fine now. Thanks a lot!
